migration.go

```diff
@@ -124,6 +146,26 @@ func (m *Migration) run(db *sql.DB, direction bool) error {
 	return nil
 }
 
+func insertOrDeleteVersionNoTx(db *sql.DB, direction bool, version int64) error {
+	if direction {
+		if _, err := db.Exec(GetDialect().insertVersionSQL(), version, direction); err != nil {
+			return err
+		}
+	}
```

Comment:
god help me how much time did I spent to debug this line
add `return nil` here

Reply:
after 153 line*

Reply:
Sorry about that...

Reply:
don't worry, at least it's a useful review :)